tutorials/setting-up-an-android-development-environment-on-compute-engine/index.md

```diff
@@ -32,11 +32,6 @@ This tutorial uses billable components of Google Cloud, including Compute Engine
 Use the [pricing calculator](https://cloud.google.com/products/calculator/) to generate a cost estimate based on your projected usage. New Google Cloud
 users might be eligible for a [free trial](https://cloud.google.com/free-trial).
 
-If you follow this tutorial, the costs are estimated as follows, depending on the operating system that you use:
-
-* [Linux VMs](https://cloud.google.com/products/calculator#id=ca135004-465c-4a43-bc5b-701af07df644)
-* [Windows VMs](https://cloud.google.com/products/calculator#id=eff1ebe1-1ed8-4475-a5f2-d07519fb5883)
```

Comment:
@ToddKopriva The following link works for me, could you do a check again.

Reply:
Yes, I'll check again as I make my second pass.

Reply:
I just checked, and the Linux link does go to a pricing calculator that shows a Linux OS, but the Windows link does not go to a pricing calculator that shows Windows Server.

Please re-create the pricing calculator links and modify the pull request with the new links.

Reply:
@ToddKopriva Thanks for letting me know about that, it must be a mistake from my end, I will fix it.